DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -37,6 +37,9 @@ public enum SessionReplay {
                 description: "Datadog SDK must be initialized before calling `SessionReplay.enable(with:)`."
             )
         }
+        guard configuration.replaySampleRate > 0 else {
+            return
```

Comment:
question/ Doesn't this mean SR usage won't be reliably measured through RUM Telemetry once configuration telemetry is finally added to `DatadogSessionReplay`? Who will we pass the [`session_replay_sample_rate`](https://github.com/DataDog/rum-events-format/blob/83f8760b46e9a117b5975cfb592b1803d643ee3e/schemas/telemetry/configuration-schema.json#L64-L70) to core if the feature is not allocated? Note that `session_replay_sample_rate: 0` is valuable information we want to collect.

Reply:
We can report the telemetry before the `guard`, not necessarily in the feature init 👍